lib/src/DatePicker/DatePicker.jsx

```diff
@@ -24,13 +24,15 @@ export class DatePicker extends PureComponent {
     renderDay: PropTypes.func,
     utils: PropTypes.object.isRequired,
     shouldDisableDate: PropTypes.func,
+    allowKeyboardControl: PropTypes.bool,
   }
 
   static defaultProps = {
     minDate: '1900-01-01',
     maxDate: '2100-01-01',
     disablePast: false,
     disableFuture: false,
+    allowKeyboardControl: false,
```

Comment:
Actually its not a breaking change - because its enabled in the DatePickerWrapper. 
The logic is - if you are using default wrapper with modal you have events disabled. If you are using some internal component - you got its disabled

Reply:
:D I think we should rename them in the next iteration